class.jetpack-xmlrpc-server.php

```diff
@@ -75,7 +75,27 @@ function bootstrap_xmlrpc_methods() {
 	}
 
 	function authorize_xmlrpc_methods() {
-		return array( 'jetpack.remoteAuthorize' => array( $this, 'remote_authorize' ) );
+		return array(
+			'jetpack.remoteAuthorize' => array( $this, 'remote_authorize' ),
+			'jetpack.activateManage'    => array( $this, 'activate_manage' ),
+		);
+	}
+
+	function activate_manage( $request ) {
+		foreach( array( 'secret', 'state' ) as $required ) {
+			if ( ! isset( $request[ $required ] ) || empty( $request[ $required ] ) ) {
+				return $this->error( new Jetpack_Error( 'missing_parameter', 'One or more parameters is missing from the request.', 400 ) );
```

Comment:
These messages will not be shown to a user. The will be read by the wpcom API and converted to more of a catch all message for failures.